src/pages/Admin/Charity/Settings/ReceiptMsg.tsx

```diff
@@ -10,7 +10,10 @@ export default function ReceiptMsg() {
     <div>
       <Field<FV, "textarea">
         type="textarea"
-        classes="field-admin [&_[data-error]]:-bottom-4"
+        classes={{
+          container: "field-admin [&_[data-error]]:-bottom-4",
+          label: "text-base font-medium",
```

Comment:
```suggestion

```
to indicate that this UI block is separate from the other block (check field - also `font-medium`)